mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -241,8 +243,62 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
     }
   ]
 
+  // Asset sort bottom sheet list
+  const assetValue = 'Asset value'
+  const [assetSortType, setAssetSortType] = useState(assetValue) // to display selected sorted type text
+  const [isSorted, setIsSorted] = useState<boolean>(false) // to display acsending/descending icon
+  const [hideIcon, setHideIcon] = useState(false)
+  const [showAssetSortBottomSheet, setShowAssetSortBottomSheet] = useState(false)
+  const modifiedDenominationCurrency = useMemo(() => denominationCurrency === 'USDT' ? 'USD' : denominationCurrency, [denominationCurrency])
+  const sortTokensAssetOnType = useCallback((assetSortType: string): BalanceRowToken[] => {
+    // from assetSortList
+    switch (assetSortType) {
+      case (`Highest ${modifiedDenominationCurrency} value`):
```

Comment:
Can the case be enum instead? Dynamic case seems weird

Reply:
Updated!